### PR TITLE
[SDK-76] Add account linking functionality

### PR DESCRIPTION
**Pull request by HarrisonHough**
## Description

-   This PR adds the functionality required to pass the login token into webview setup functions and build a URL with the logintoken param (if provided
- this enables devs to trigger auto login, they just need to implement the process of requesting this token from there back end (it can't be done client side) 

#### Added

- loginToken param to webview setup function
- login token param to BuildURL process
- ReloadUrlWithToken to trigger reload without reinitialization.

## How to Test

-   Run webview on a device and check logs for authorized event after logging in 

## Checklist

-   [ ] Tests written or updated for the changes.
-   [ ] Documentation is updated.
-   [ ] Changelog is updated.
-   [ ] QA Testing is updated.

## QA Testing

-   Mention any useful test cases for this feature and add to QA Testing documentation.

## Details

-   If more details are necessary to support the description with images and videos add them here.

[SDK-76]: https://ready-player-me.atlassian.net/browse/SDK-76?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Runtime/WebViewPanel.cs [hunk 1]
```diff
@@ -51,7 +51,7 @@ public void LoadWebView()
         /// <summary>
         /// Initializes the WebView if it is not already and enables the WebView window.
         /// </summary>
-        private void InitializeAndShowWebView()
+        private async void InitializeAndShowWebView(string loginToken = "")
```
Comment (srcnalt) [inline Runtime/WebViewPanel.cs:54]: We do not await for any async method in here, was there a reason to add this?

**Comment by rYuuk** [comment]
I that you are passing loginToken in all public functions, wouldn't it be better to set it once in another function/property and then use it everywhere?

File: Runtime/WebViewPanel.cs (hunk 1 above)
Comment (HarrisonHough) [reply Runtime/WebViewPanel.cs:54]: Ah  yep good catch, had it there initially as I was adding the login token request client side, which had an await. 
removed now

**Comment by HarrisonHough** [comment]
> I see that you are passing loginToken in all public functions, wouldn't it be better to set it once in another function/property and then use it everywhere?

The login token only lasts 15 seconds so it needs to be used immediately

File: Runtime/Data/UrlConfig.cs [hunk 2]
```diff
@@ -30,17 +33,21 @@ public class UrlConfig
 
         [Tooltip("Skip body type selection and create avatars with selected body type. Ignored if Quick Start is checked.")]
         public BodyType bodyType = BodyType.Selectable;
-
+
```
Comment (gadamyan) [inline Runtime/Data/UrlConfig.cs:36]: Not sure if you comment on indentation issues

File: Runtime/WebViewPanel.cs [hunk 3]
```diff
@@ -77,6 +76,13 @@ private void InitializeAndShowWebView()
             }
         }
 
+        public void ReloadWithLoginToken(string loginToken = "")
+        {
+            urlConfig ??= new UrlConfig();
+            var url = urlConfig.BuildUrl(loginToken);
+            webViewObject.LoadURL(url);
```
Comment (gadamyan) [inline Runtime/WebViewPanel.cs:83]: Just an Idea, I have no idea about the browser stuff, Is browser cache cleaning needed here?

**Review by rYuuk — APPROVED** [review]
🟢 Great job, works as expected.

**Comment by srcnalt** [comment]
Hi @Thaina this is an early feature which is not yet publicly available. We will share more details when it is finalized. Meanwhile you can join our Discord to be notified of latest news https://discord.gg/readyplayerme

**Comment by Thaina** [comment]
I have join the discord but there was no answer about this feature

**Pull request merged**